runtime/src/lib.rs

```diff
@@ -811,35 +812,14 @@ mod custom_migration {
             let accounts: Vec<AccountId> = Self::get_accounts();
             staking_upgrade::<Runtime>();
             accounts_upgrade::<Runtime>(accounts);
+            Anchor::migrate_anchors();
```

Comment:
Since this doesn't require any migration dependencies, let move this to `on_runtime_upgrade` in the anchor module ?